### PR TITLE
Visitors can also inform about the preferred location

### PR DESCRIPTION
**Pull request by lukeperrin10**
PT-Story: https://www.pivotaltracker.com/story/show/178413034

User Story

```
As a user
In order to receive a better qualified service
I want to provide some additional info about my inquiry

```

Changes proposed in this pull request:

- Dropdown menu for area selection in Gothenburg 
- Adds styling to menu 
- updates API address to reflect the new 

What I have learned working on this feature:

- More knowledge in styling through scss

Packages/Gems added

- react-responsive-select

Screenshots (Client)

Kazam_screenshot_00000

Kazam_screenshot_00001

http://recordit.co/JbsfmN1NcN

### CODE REVIEW

**Review by DavDan1 — APPROVED** [review]

**Review by arunbhalli — APPROVED** [review]

File: src/components/custom/CustomSelectInput.jsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  {
+    value: 'Majorna',
+    text: 'Majorna',
+    markup: <MultiSelectOptionMarkup text='Majorna' />,
+  },
+  {
+    value: 'Avenyn',
+    text: 'Avenyn',
+    markup: <MultiSelectOptionMarkup text='Avenyn' />,
+  },
```
Comment (offcast-dev) [inline src/components/custom/CustomSelectInput.jsx]: Extract to separate file

File: src/components/custom/CustomSelectInput.jsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        markup: <MultiSelectOptionMarkup text='Alla' />,
+      }
+    } else {
+      locations[0] = {
+        value: 'null',
+        text: 'Any',
+        markup: <MultiSelectOptionMarkup text='Any' />,
+      }
+    }
+    // eslint-disable-next-line
```
Comment (1CIM) [inline src/components/custom/CustomSelectInput.jsx:35]: is this needed now?

File: src/locales/en.js [hunk 3]
```diff
@@ -16,17 +16,21 @@ const en = {
...
+      peers: 'Would you like to share office with others from your industry?',
+      location: 'Where would you like to be situated?',
     },
     answer: {
-      email: 'My email is: ',
-      size: 'We are a group of: ',
-      officeType: 'We need: ',
+      email: 'My email is... ',
+      size: 'We are a group of... ',
+      officeType: 'We need an... ',
```
Comment (offcast-dev) [inline src/locales/en.js]: are you shure about three dots ? If you don't like: maby we should get rid of both

**Review by offcast-dev — CHANGES_REQUESTED** [review]
locations should be extracted

File: src/locales/se.js [hunk 4]
```diff
@@ -19,19 +19,22 @@ const se = {
...
-      officeType: 'Vi behöver: ',
+      email: 'Min email är... ',
+      size: 'Vi är en grupp av... ',
+      officeType: 'Vi behöver... ',
       officeLable: 'Kontor',
       openSpaceLable: 'Öppet utrymme',
-      name: 'Det heter: ',
-      companyPlaceholder: 'Mitt företag',
+      name: 'Det är... ',
+      companyPlaceholder: 'MyCoolCompany',
```
Comment (1CIM) [inline src/locales/se.js]: Not Swedish but i think its fine but can you try to come up with a Swedish phrase here?

File: src/styles/questionnaire.scss [hunk 5]
```diff
@@ -200,3 +201,76 @@
     }
   }
 }
+
+.location-question-container {
+  .answer-form-container {
+    display: flex;
+
+    .rrs__multiselect-label__badge {
```
Comment (1CIM) [inline src/styles/questionnaire.scss:209]: what is this rrs stuff?

File: src/styles/variables.css [hunk 6]
```diff
@@ -1 +1 @@
-/* No CSS */
\ No newline at end of file
+/* No CSS *//*# sourceMappingURL=variables.css.map */
```
Comment (1CIM) [inline src/styles/variables.css:1]: ? what this do? auto gen?

**Review by 1CIM — COMMENTED** [review]
Feels like the prettier had a problem in this pr. Mid giving it a look?

File: src/components/Inquiry/Answer.jsx [hunk 7]
```diff
@@ -1,60 +1,69 @@
...
-    event.preventDefault()
+    event.preventDefault();
     if (filled) {
       store.dispatch({
         type: 'SWITCH_ANSWER_FILLED_STATUS',
         payload: { key: questionKey, filled: false },
-      })
+      });
     } else {
+      animateScroll.scrollMore(isSmall ? 280 : 200);
```
Comment (offcast-dev) [inline src/components/Inquiry/Answer.jsx:31]: are all containers the same size for scroll? can we take the height of the container and put it into variable? Or scroll **to** specific element

File: src/styles/questionnaire.scss (hunk 5 above)
Comment (lukeperrin10) [reply src/styles/questionnaire.scss:209]: It's the name of the class generated by the module we are using to fix the select menu

File: src/styles/variables.css (hunk 6 above)
Comment (lukeperrin10) [reply src/styles/variables.css:1]: yeah auto generated

File: src/components/Inquiry/Answer.jsx (hunk 7 above)
Comment (lukeperrin10) [reply src/components/Inquiry/Answer.jsx:31]: I mean we could, but might be over complicating it for 8 questions.

File: src/locales/se.js (hunk 4 above)
Comment (lukeperrin10) [reply src/locales/se.js]: Can take a look at it.

File: src/locales/en.js (hunk 3 above)
Comment (lukeperrin10) [reply src/locales/en.js]: Removed

File: src/components/custom/CustomSelectInput.jsx (hunk 2 above)
Comment (lukeperrin10) [reply src/components/custom/CustomSelectInput.jsx:35]: Yes or we get a compiler warning which will mess up netlify

File: src/locales/se.js (hunk 4 above)
Comment (lukeperrin10) [reply src/locales/se.js]: Changed

File: src/components/custom/CustomSelectInput.jsx (hunk 1 above)
Comment (lukeperrin10) [reply src/components/custom/CustomSelectInput.jsx]: Extracted

**Review by offcast-dev — APPROVED** [review]

**Pull request merged**